docs/design/analytics.md

```diff
@@ -25,94 +25,86 @@ One dimension per flag, and although technically there can be an overlap between
 1. Use the ID of the dimension as the `x-user-analytics` value in the `schema.json` file.
-1. Add a new row to the table below in the same PR as the one adding the dimension to the code.
-1. New dimension PRs need to be approved by the tooling and DevRel leads.
-   **This is not negotiable.**
+1. New dimension and metrics PRs need to be approved by the tooling lead and require a new (http://go/launch)[Launch].
+
+### Deleting a dimension or metic.
+1. Archive the dimension and metric in (https://analytics.google.com/)[Google Analytics].
+
 
 **DO NOT ADD `x-user-analytics` FOR VALUES THAT ARE USER IDENTIFIABLE (PII), FOR EXAMPLE A
 PROJECT NAME TO BUILD OR A MODULE NAME.**
 
-Note: There's a limit of 20 custom dimensions.
+### Limits
+| Item                           	| Standard property limits 	|
+|--------------------------------	|--------------------------	|
+| Event-scoped custom dimensions 	| 50                       	|
+| User-scoped custom dimensions  	| 25                       	|
+| All custom metrics             	| 50                       	|
 
-### List Of All Dimensions
+### List of User Custom Dimensions
+
+<!--USER_DIMENSIONS_TABLE_BEGIN-->
+| Name | Parameter | Type |
+|:---:|:---|:---|
+| Command | `ep.ng_command` | `string` |
+| SchematicCollectionName | `ep.ng_schematic_collection_name` | `string` |
+| SchematicName | `ep.ng_schematic_name` | `string` |
+| Standalone | `ep.ng_standalone` | `string` |
+| Style | `ep.ng_style` | `string` |
+| Routing | `ep.ng_routing` | `string` |
+| InlineTemplate | `ep.ng_inline_template` | `string` |
+| InlineStyle | `ep.ng_inline_style` | `string` |
+| BuilderTarget | `ep.ng_builder_target` | `string` |
+| Aot | `ep.ng_aot` | `string` |
+| Optimization | `ep.ng_optimization` | `string` |
+<!--USER_DIMENSIONS_TABLE_END-->
+
+### List of Event Custom Dimensions
 
 <!--DIMENSIONS_TABLE_BEGIN-->
-| Id | Flag | Type |
+| Name | Parameter | Type |
 |:---:|:---|:---|
-| 1 | `CPU Count` | `number` |
-| 2 | `CPU Speed` | `number` |
-| 3 | `RAM (In GB)` | `number` |
-| 4 | `Node Version` | `number` |
-| 5 | `Flag: --style` | `string` |
-| 6 | `--collection` | `string` |
-| 7 | `Flag: --strict` | `boolean` |
-| 8 | `Angular CLI Major Version` | `string` |
-| 9 | `Flag: --inline-style` | `boolean` |
-| 10 | `Flag: --inline-template` | `boolean` |
-| 11 | `Flag: --view-encapsulation` | `string` |
-| 12 | `Flag: --skip-tests` | `boolean` |
-| 13 | `Flag: --aot` | `boolean` |
-| 14 | `Flag: --minimal` | `boolean` |
-| 15 | `Flag: --standalone` | `boolean` |
-| 16 | `Flag: --optimization` | `boolean` |
-| 17 | `Flag: --routing` | `boolean` |
-| 18 | `Flag: --skip-import` | `boolean` |
-| 19 | `Flag: --export` | `boolean` |
-| 20 | `Build Errors (comma separated)` | `string` |
+| Command | `ep.ng_command` | `string` |
+| SchematicCollectionName | `ep.ng_schematic_collection_name` | `string` |
+| SchematicName | `ep.ng_schematic_name` | `string` |
+| Standalone | `ep.ng_standalone` | `string` |
```

Comment:
This collects only `--standalone` flag, right?

Reply:
Yes